python/niizarr/_zarr2nii.py

```diff
@@ -96,19 +108,33 @@ def zarr2nii(inp, out=None, level=0):
 
     # reorder/reshape array as needed
     array = dask.array.from_zarr(inp[f'{level}'])
+
     if array.ndim == 5:
         array = array.transpose([4, 3, 2, 0, 1])
+        assert inp[level].attrs['_ARRAY_DIMENSIONS'] == ['time', 'channel', 'z', 'y', 'x']
```

Comment:
I am not sure the `'_ARRAY_DIMENSIONS'` is always set (I believe it's for back-compatibility with some older xarray representation). It's more robust to look at the axes in `group.attrs['multiscales'][0]['axes']` (at the zarr group level)